Quickstart doesn't work out of the box, defeating the point

If I copy and paste the quickstart example and try to compile it, it doesn't work

please provide sufficient information to make the quickstart actually "quick"

```
   Compiling testt v0.1.0 (/tmp/testt)
error[E0433]: failed to resolve: use of undeclared type or module `async_std`
 --> src/main.rs:7:3
  |
7 | #[async_std::main] // or #[tokio::main]
  |   ^^^^^^^^^ use of undeclared type or module `async_std`

error[E0277]: the trait bound `std::env::VarError: sqlx::error::DatabaseError` is not satisfied
  --> src/main.rs:12:43
   |
12 |         .connect(&env::var("DATABASE_URL")?).await?;
   |                                           ^ the trait `sqlx::error::DatabaseError` is not implemented for `std::env::VarError`
   |
   = note: required because of the requirements on the impl of `std::convert::From<std::env::VarError>` for `sqlx::Error`
   = note: required by `std::convert::From::from`

error[E0277]: `main` has invalid return type `impl std::future::Future`
 --> src/main.rs:8:20
  |
8 | async fn main() -> Result<(), sqlx::Error> {
  |                    ^^^^^^^^^^^^^^^^^^^^^^^ `main` can only return types that implement `std::process::Termination`
  |
  = help: consider using `()`, or a `Result`

error[E0752]: `main` function is not allowed to be `async`
 --> src/main.rs:8:1
  |
8 | async fn main() -> Result<(), sqlx::Error> {
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `main` function is not allowed to be `async`

error: aborting due to 4 previous errors

Some errors have detailed explanations: E0277, E0433, E0752.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `testt`.

To learn more, run the command again with --verbose.
```

--- a/README.md
+++ b/README.md
@@ -161,7 +161,8 @@ sqlx = { version = "0.4.0-beta.1", default-features = false, features = [ "runti
 ## Usage
 
 ### Quickstart
-```
+
+```toml
 [dependencies]
 sqlx = { version = "0.4.0-beta.1", features = [ "postgres" ] }
 async-std = { version = "1.6", features = [ "attributes" ] }

--- a/README.md
+++ b/README.md
@@ -161,6 +161,11 @@ sqlx = { version = "0.4.0-beta.1", default-features = false, features = [ "runti
 ## Usage
 
 ### Quickstart
+```
+[dependencies]
+sqlx = { version = "0.4.0-beta.1", features = [ "postgres" ] }
+async-std = { version = "1.6", features = [ "attributes" ] }
+```
 
 ```rust
 use std::env;
@@ -174,7 +179,7 @@ async fn main() -> Result<(), sqlx::Error> {
     // Create a connection pool
     let pool = PgPoolOptions::new()
         .max_connections(5)
-        .connect(&env::var("DATABASE_URL")?).await?;
+        .connect("postgres://postgres:password@localhost/test").await?;
 
     // Make a simple query to return the given parameter
     let row: (i64,) = sqlx::query_as("SELECT $1")

--- a/README.md
+++ b/README.md
@@ -168,8 +168,6 @@ async-std = { version = "1.6", features = [ "attributes" ] }
 ```
 
 ```rust
-use std::env;
-
 use sqlx::postgres::PgPoolOptions;
 // use sqlx::mysql::MySqlPoolOptions;
 // etc.

